scheduler-driver/src/test/java/gov/nasa/jpl/aerie/scheduler/TestApplyWhen.java

```diff
@@ -204,7 +209,7 @@ public void testRecurrenceBabyWindow() {
       logger.debug(a.startOffset().toString());
     }
 
-    assertFalse(TestUtility.activityStartingAtTime(plan,Duration.of(1, Duration.SECONDS), activityType));
+    assertTrue(TestUtility.activityStartingAtTime(plan,Duration.of(1, Duration.SECONDS), activityType));
```

Comment:
Why is this test now expected to return the inverse?

Reply:
The test was wrong in the first place. Below a detailed explanation:
The plan horizon ranges from [0,20).
The recurrent activities can be placed inside the following window: [1,2)
The activities instantiating the goal have a duration of 1
The activities should repeat every 5 time units

It is therefore possible to place an activity in time 1, with a duration of 1
Graphically
    // RECURRENCE WINDOW: [+----+----+----+----]
    // GOAL WINDOW:              [+-------------------]
    // RESULT:                            [+-------------------]